Server/src/models/DatasetModels/DatasetApprovalModel.ts

```diff
@@ -70,9 +71,12 @@ export class DatasetApprovalModel {
     }
 
     async updateDatasetComments(datasetId: number, datasetCommentsToAppend?: string) {
-        if (datasetCommentsToAppend !== null && datasetCommentsToAppend !== undefined) {
+        if (datasetCommentsToAppend !== undefined && datasetCommentsToAppend !== "undefined" && datasetCommentsToAppend !== null) {

```

Comment:
i dont think its possible for it to be a string of "undefined", just if(datasetCommentToAppend) should suffice

Reply:
It is in fact possible to get a string of "undefinied" and that was the primary issue to solve.